src/teststand.cpp

```diff
@@ -167,7 +171,10 @@ bool Teststand::acquire_sensors()
 bool Teststand::send_target_joint_torque(
     const Eigen::Ref<Vector2d> target_joint_torque)
 {
-  joints_.set_torques(target_joint_torque);
+  Vector2d ctrl_torque = target_joint_torque;
+  ctrl_torque = ctrl_torque.array().min(max_joint_torques_);
+  ctrl_torque = ctrl_torque.array().max(- max_joint_torques_);
```

Comment:
Can you do 

```
ctrl_torque.array().min(max_joint_torques_).max(- max_joint_torques_);
```
to safe one line?

Does Eigen have an API for clip, which is like `ctrl_torque.array().clip(-max_joint_torques_, max_joint_torques_)`?

Reply:
First comment I am actually not sure if this works...

Second comments, I looked around but their do not seem to be the case.

Reply:
you may use [this](https://github.com/open-dynamic-robot-initiative/mpi_cpp_tools/blob/master/include/mpi_cpp_tools/math.hpp#L25) function from mpi_cpp_tools :).

Reply:
Actually I would prefer a:
`clamp(Vector in, Vector lb, Vector ub)`
and not a 
`clamp(Vector in, double lb, double ub)`

Reply:
ok, maybe you could just add it to mpi_cpp_tools

Reply:
Ok, but this will be done in another PR.